include/crypto/x509.h

```diff
@@ -80,6 +80,8 @@ struct X509_req_st {
 
     /* Set on live certificates for authentication purposes */
     ASN1_OCTET_STRING *distinguishing_id;
+    OSSL_LIB_CTX *libctx;
+    char *propq;
```

Comment:
Is it worthwhile considering this as a `const char *` ?
Yeah, I know it means a cast on free which is why I'm not too fussed.

Reply:
This is like this in every other struct - so if you wanted to do that it should be another PR.. I don't think it is necessary.

Reply:
Okay, ignore my ravings.